packages/vats/src/lib-chainStorage.js

```diff
@@ -113,3 +113,46 @@ export async function makeStorageNode(chainStorage, childName) {
   return storageNode;
 }
 harden(makeStorageNode);
+
+/**
+ * For testing, creates a chainStorage root node over an in-memory map
+ * and exposes both the map and the sequence of received messages.
+ *
+ * @param {string} rootPath
+ */
+export const makeFakeStorageKit = rootPath => {
```

Comment:
I like the name, but could we move this function into another file? Something with "test" in the path would be ideal, but I'd settle for a dedicated "fake-chainStorage" if something about the former doesn't work.

Reply:
`test/` shouldn't be exported but I'll try something like `TestUtils`